ci/travis-install-librpma.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+# 11.02.2021 Merge pull request #866 from ldorau/rpma-mmap-memory-for-rpma_mr_reg-in-rpma_flush_apm_new
+LIBRPMA_VERSION=fbac593917e98f3f26abf14f4fad5a832b330f5c
+
+WORKDIR=$(pwd)
+
+# install librpma
+git clone https://github.com/pmem/rpma.git
```

Comment:
Maybe grab the tarball like you do below? I guess technically I prefer to see curl used even if it needs more parameters :-)